src/main/java/ch/naviqore/gtfs/schedule/model/GtfsSchedule.java

```diff
@@ -24,6 +24,9 @@ public class GtfsSchedule {
     private final Map<String, Stop> stops;
     private final Map<String, Route> routes;
     private final Map<String, Trip> trips;
+    private final boolean hasAccessibilityInformationForStops;
```

Comment:
How often do we need this information?

If we only need this once, when configuring the raptor, we could also just check this with a Stream.anyMatch on the fly.



Maybe also rename the variables, since the getter names are a bit ugly:

![image](https://github.com/user-attachments/assets/ad28ad47-f883-4718-b5bb-f540fc0489d8)

Reply:
Renamed the attributes in 6470745.

I would keep them in the schedule, because I think this are attributes that belong to the schedule and do not cost anything to maintain. For example if we update the schedule and the newer version newly contains attributes or has them removed, this way we can directly reference the three booleans on the schedule. Maybe also other use cases.